cms/envs/bok_choy.py

```diff
@@ -25,6 +25,7 @@
 # and (b) that the files are the same in Jenkins as in local dev.
 os.environ['SERVICE_VARIANT'] = 'bok_choy_docker' if 'BOK_CHOY_HOSTNAME' in os.environ else 'bok_choy'
 os.environ['CONFIG_ROOT'] = path(__file__).abspath().dirname()
+os.environ['STUDIO_CFG'] = str.format("{0}/{1}.yml", os.environ['CONFIG_ROOT'], os.environ['SERVICE_VARIANT'])
```

Comment:
prefer `"{named_var1}/{named_var2}.yml".format( named_var1 = os.environ['CONFIG_ROOT'], named_var2 = os.environ['SERVICE_VARIANT'])` for readability of formatted strings

Reply:
Ah cool - I didn't know you could do this, ty.